api.htmlui/test/unit/src/org/netbeans/modules/htmlui/AnnotationsTest.java

```diff
@@ -0,0 +1,58 @@
+/**
```

Comment:
Although it's a trivial thing, currently `/*` is used (#282).

Reply:
Thanks for spotting this - a copy&paste&merge error, I guess. Fixed in f59e878